databricks_blueprints/helpers.py

```diff
@@ -0,0 +1,90 @@
+import sys
+import requests
+import shipyard_utils as shipyard
+try:
+    import errors
+except BaseException:
+    from . import errors
+
+# Create Artifacts folders
+base_folder_name = shipyard.logs.determine_base_artifact_folder(
+    'databricks')
+artifact_subfolder_paths = shipyard.logs.determine_artifact_subfolders(
+    base_folder_name)
+shipyard.logs.create_artifacts_folders(artifact_subfolder_paths)
+
```

Comment:
Smart! I really like this setup where (I'm assuming) every script imports helpers which will automatically create the Artifacts folders without needing to specify it in each script.